src/pages/[guild]/activity.tsx

```diff
@@ -55,7 +55,10 @@ const ActivityLog = (): JSX.Element => {
             href={`/${urlName}/events`}
             onClick={() => {
               setEventsSeen(true)
-              captureEvent("Click on events tab", { from: "activity log" })
+              captureEvent("Click on events tab", {
+                from: "activity log",
+                guildUrl: urlName,
```

Comment:
Please use `guild: urlName`, so we can filter this event the same way we can filter other events ;)